internal/provider/models/destinations/test/elasticsearch_test.go

```diff
@@ -72,7 +72,7 @@ func TestElasticSearchDestinationResource(t *testing.T) {
 						description = "my destination description"
 						inputs      = [mezmo_http_source.my_source.id]
 						pipeline_id = mezmo_pipeline.test_parent.id
-						endpoints   = ["https://example.com"]
+						endpoints   = ["https://google.com"]
```

Comment:
Curious why this change.  `example.com` is a real domain after all.

Reply:
The domain is down hardddd

Reply:
Stupid RFC fact of the day:

[RFC 2606](https://www.rfc-editor.org/rfc/rfc2606.html) defines four TLD that are reserved and avoid network traffic if used - `.test`, `.example`, `.invalid` and `.localhost`. Maybe we should consider using `.test` if we're worried about the time it takes to resolve to an IP address or some other network activity.